### ISSUE TITLE
can not delete media from app

### ISSUE THREAD

**Issue by somethingGoneWrong** [jellyfin/jellyfin-tizen#83]
I have the latest server version and the latest git version of tizen app and when I try to delete a media from app this not work, if I try to delete the same media via web interface, jellyfin remove it correctly.
is this a know issue? where can i find the log to help you to fix this? thx

**Comment by dmitrylyzo** [comment]
Tizen app just wraps `jellyfin-web`, so it uses the same command as web (it lives in `jellyfin-apiclient-javascript`).

In Tizen 3 emulator:
`Delete media` works on `jellyfin-server_20210505.37-unstable` + latest `jellyfin-tizen` (with jellyfin/jellyfin-web@e27131edf0462c557d1c53ea4ee65fa6f4ac5b5c).
Also works on `jellyfin-server_10.7.5-1` + the same app.

It is not easy to get logs from app: https://github.com/jellyfin/jellyfin-tizen/issues/79#issuecomment-822068324
You can find the location of the server logs on the `Dashboard` home page.

When you click on `Delete media`, a message box with `OK/Cancel` buttons should appear. Did it show up?

**Comment by somethingGoneWrong** [comment]
> When you click on Delete media, a message box with OK/Cancel buttons should appear. Did it show up?
no, nothing appears; after click on delete media the menu is closed

**Comment by dmitrylyzo** [comment]
> no, nothing appears; after click on delete media the menu is closed

So, the problem is on the client side.
The same dialog is displayed when removing subtitles (external only). This also works on my TV (Tizen 4).

What version of Tizen?
Could you try to delete media using Samsung Browser (do not confirm deletion; need to check if the dialog appears)? _In this case, you will use `jellyfin-web` from the server._ It might be tricky without a keyboard.

**Comment by somethingGoneWrong** [comment]
> What version of Tizen?

server:  10.7.5
app JF:  latest git
tizen TV:  tv-2.3.0

> Could you try to delete media using Samsung Browser (do not confirm deletion; need to check if the dialog appears)?

Yes, it works; when I try to delete a video I have a dialog box from tizen sdk, that ask me if I want to delete the media with two choice [yes, no]; and delete it.
maybe the problem is the creation of a tizen modal dialog from JF app?
strange thing is the use the JF from web broswer is smoother than the JF tizen app.
here some server log cleaned: https://pastebin.com/eLQdt03v

**Comment by dmitrylyzo** [comment]
> maybe the problem is the creation of a tizen modal dialog from JF app?

Yeah, I think the same. Also Tizen 2.3 uses the old WebKit engine.
I checked the webOS 1.2 emulator (also WebKit) - it doesn't show dialog but removes media. Anyway, that's bad: missclick - and you've lost your media.
Tizen probably resolves with `false`.

> strange thing is the use the JF from web broswer is smoother than the JF tizen app.

I suspect it is using a different engine.
Could you find out user agent of Samsung Browser? For example, go to https://shaka-player-demo.appspot.com/support.html
If it contains `Chrome`, that's why it works better.

For your case, you can change https://github.com/jellyfin/jellyfin-web/blob/e27131edf0462c557d1c53ea4ee65fa6f4ac5b5c/src/components/confirm/confirm.js#L63
to `const confirm = customConfirm;`
_I checked this after I wrote. The custom dialog looks ugly - need to adapt the layout for fullscreen._

**Comment by somethingGoneWrong** [comment]
> Could you find out user agent of Samsung Browser? 

[mozilla gecko and other stuff](https://ibb.co/gytmxvY)
when I have some time I will try the modification on confirm.js file.

Another strange behavior is at the login screen, after I put the IP of the server the JF app ask me the username and password; that's ok, but when I typed the username and press OK button to the keyboard widget to close that and select the password field, the app try to connect to the JF server and not allow me to add a password, obviously it failed, my workaround is to fill the password field before the username and then after a failed login attempt fill the username field...
Is this a bug related for my old tizen sdk platform installed on TV? but this is another issue :)

**Comment by dmitrylyzo** [comment]
> mozilla gecko and other stuff

Looks the same as app (WebKit). Then I don't know why it works smoother.

I always press `Back` to close the on-screen keyboard.
`OK` on the on-screen keyboard is like `OK` on the remote, i.e. `Enter`, which submits the login form.
